### PR TITLE
Cl audits

### PR DESCRIPTION
**Pull request by linuskendall**
Adds a new audit table for decompression tasks. Also adds more logging.

### CODE REVIEW

File: nft_ingester/src/program_transformers/bubblegum/mod.rs [hunk 1]
```diff
@@ -32,56 +32,76 @@ where
     T: ConnectionTrait + TransactionTrait,
 {
     let ix_type = &parsing_result.instruction;
-    match ix_type {
+
+    // @TODO this would be much better served by implemneting Debug trait on the InstructionName
```
Comment (fanatid) [inline nft_ingester/src/program_transformers/bubblegum/mod.rs:36]: you already can remove `debug!` from the match here and use it after
`debug!("{ix_str} instruction:)`

File: nft_ingester/src/program_transformers/bubblegum/db.rs [hunk 2]
```diff
@@ -58,6 +64,10 @@ where
             leaf_idx: Set(leaf_idx),
             ..Default::default()
         };
+
+        let mut audit_item : cl_audits::ActiveModel = item.clone().into();
+        audit_item.tx = Set(String::from(txn_id));
```
Comment (fanatid) [inline nft_ingester/src/program_transformers/bubblegum/db.rs:69]:
```suggestion
        audit_item.tx = Set(txn_id.to_string());
```

**Review by fanatid — APPROVED** [review]
table from what I know about sea orm looks good for me

File: nft_ingester/src/program_transformers/bubblegum/mod.rs (hunk 1 above)
Comment (linuskendall) [reply nft_ingester/src/program_transformers/bubblegum/mod.rs:36]: Leaving the info! only

**Pull request merged**